docs/pages/distribution/publishing-websites.mdx

```diff
@@ -55,27 +82,25 @@ Use [Serve CLI](https://www.npmjs.com/package/serve) to quickly test locally how
 
 <Tabs>
 
-<Tab label="Expo Router">
-
-If you're using [static rendering](/router/reference/static-rendering), then you can serve the static bundle with the following command:
-
-<Terminal cmd={['$ npx serve dist']} />
-
-Otherwise, use the `--single` flag to redirect all requests to the single **dist/index.html** file.
+<Tab label="single">
 
 <Terminal cmd={['$ npx serve dist --single']} />
 
-Open [`http://localhost:5000`](http://localhost:5000) to see your project in action. This method is **HTTP only**, so permissions, camera, location, and many other secure features won't work.
+Open [`http://localhost:5000`](http://localhost:5000) to see your project in action. This method is **HTTP only**, so permissions, camera, location, and many other secure features will not work.
 
 </Tab>
 
-<Tab label="webpack">
+<Tab label="static">
+
+<Terminal cmd={['$ npx serve dist']} />
 
-Run the following command to serve the single-page application:
+Open [`http://localhost:5000`](http://localhost:5000) to see your project in action. This method is **HTTP only**, so permissions, camera, location, and many other secure features will work.
+
+</Tab>
```

Comment:
Let's move this after the server since it seems to be the pattern for this whole document (and we should follow it even though it might give our users a tiny but uniformed experience).